tests/commons/glob/test_glob.py

```
@@ -0,0 +1,159 @@
+from sema.commons.glob import (
+    GlobMatchVisitor,
+    getMatchingGlobPaths,
+    pathMatchesGlob,
+    visitGlobPaths,
+)
+
+
+def test_getMatchingGlobPaths() -> None:
+    assert getMatchingGlobPaths
+    root = TEST_FOLDER / "data/glob"
+    assert root.exists() and root.is_dir()
```

Ruff (0.8.2)

Assertion should be broken down into multiple parts

Break down assertion into multiple parts

(PT018)